harvester/harvester
# Bump Kubevirt to v0.49.0

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Bump Kubevirt to a newer version of some dependency features, e.g. https://github.com/harvester/harvester/issues/574

**Solution:**
Bump Kubevirt to v0.49.0


**Related Issue:**
https://github.com/harvester/harvester/issues/1783

**Test plan:**
- Verify the CI is passed.
- Run `make build-iso` and has no blockers.
- Install the Harvester with the local ISO build and verify all of the system components are running with `health` status.
- Run regular regress tests, e.g upload an image, create a VM and create a Volume, etc.
- Note: upgrade from v1.0.0 to the newer version will need to be verified via https://github.com/harvester/harvester/issues/1861

Note: please ignore the CodeFactor CI because of an existing format issue.
